vulnz/src/docker/scripts/mirror.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+echo "Updating..."
+
+java $JAVA_OPT -jar /usr/local/bin/vulnz cve --cache --directory /usr/local/apache2/htdocs
```

Comment:
By default, an NVD_API_KEY is not provided. I'd suggest for the docker image we just don't use one as it is only really needed on the first run. Instead, just increase the delay:

```suggestion
java $JAVA_OPT -jar /usr/local/bin/vulnz cve  --delay=10000 --cache --directory /usr/local/apache2/htdocs
```

Yes, this will slow it down for those that did supply an API key... maybe we need to check for the presence of the API key. thoughts?

Reply:
Thanks for the feedback

What i did was

```
- `NVD_API_KEY`: define your API key
- `DELAY`: override the delay. If you do not set an API KEY, the delay will be `10000`
```

So if the API key is missing, we set a delay of 10000, otherwise it will not be set (should we set a specific value with an API key?

In addition, the user can enforce a DELAY value when hey sets the env var.

Is that good enough in your book?

Reply:
good enough! Thanks